hr_timesheet_sheet/static/src/scss/backend.scss

```diff
@@ -1,9 +1,46 @@
-.hr_timesheet_sheet_summary .o_field_x2many_2d_matrix table {
-    tbody td {
-        text-align: left;
+.hr_timesheet_sheet_summary .o_field_x2many_2d_matrix {
+    .table-responsive {
+        max-height: 450px;
+        overflow-y: auto;
     }
 
-    tfoot td {
-        text-align: left;
+    table {
+        tbody {
+            td {
+                text-align: left;
+
+                &.row-total {
+                    position: sticky;
+                    right: 0;
+                    border-left-width: 1px;
+                    border-left-color: #ccc;
```

Comment:
Isn't there a color for that in theme definition?

Reply:
Updated, thanks!